transforms/csdl/transform.ps1

```diff
@@ -14,6 +14,8 @@ param (
     $removeCapabilityAnnotations = $true,
     [bool]
     $addInnerErrorDescription = $false
```

Comment:
I think we are also missing a trailing comma after this parameter.

Reply:
Resolved.